src/storage/storage.repository.ts

```diff
@@ -32,4 +32,27 @@ export class StorageRepository {
   async delete(id: string) {
     return this.prismaService.storage.delete({ where: { id } });
   }
+
+  async assignIngredient(storageId: string, ingredientId: string) {
```

Comment:
Please remove these methods when we add entity, because repository should not have such logic